Source/WPF/MyMoney/Properties/AssemblyInfo.cs

```diff
@@ -46,9 +45,9 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: NeutralResourcesLanguage("")]
+[assembly: NeutralResourcesLanguageAttribute("")]

```

Comment:
Including the "Attribute" suffix in custom attributes is not required.

Reply:
This was altered by the build tools, it was emitting a build warning

Reply:
oh, weird, good to know thanks.